pages/index.jsx

```diff
@@ -12,6 +15,41 @@ const Header = styled.header`
   background: white;
 `;
 
+const Container = styled.div`
+  width: 1280px;
+  height: 80px;
+  padding: 0 2rem;
+  display: flex;
+  justify-content: space-between;
```

Comment:
Interestingly this was enough when I built the header to make it pixel-perfect (which I didn't expect actually). No idea what the difference is in your code. But one way to make this close to pixel-perfect is to wrap the logo and the link in a `div` and set `flex: 1` on those divs. This way they expand to take up equal space and the nav links are positioned in the middle.

![image](https://user-images.githubusercontent.com/9722792/191945166-52113ecc-79f6-4f18-a1b2-7881d6f3c35e.png)